### PR TITLE
Build triggers next job in the workflow

### PR DESCRIPTION
**Pull request by FenrirUnbound**
When a build has succeeded -- and is updated as such -- we look up the next build in the workflow and trigger that build prior to returning.

### CODE REVIEW

File: plugins/builds/update.js [hunk 1]
```diff
@@ -66,10 +67,37 @@ module.exports = () => ({
...
+                        const nextJobName = workflow[workflowIndex + 1];
 
-                    // Update the model in datastore
-                    return build.update();
+                        return jobFactory.get({
+                            name: nextJobName,
+                            pipelineId: pipeline.id
+                        }).then((nextJobToTrigger) => buildFactory.create({
+                            jobId: nextJobToTrigger.id,
+                            username
```
Comment (stjohnjohnson) [inline plugins/builds/update.js]: Should we also pass the SHA of the current build?

Comment (d2lam) [inline plugins/builds/update.js]: Sure we can

**Comment by stjohnjohnson** [comment]
Please add labels.

**Comment by stjohnjohnson** [comment]
LGTM 👍

**Comment by tkyi** [comment]
👍 LGTM
giphy-1

**Pull request merged**